src/Tests/Seam/When_receiving_messages.cs

```diff
@@ -29,10 +29,30 @@ public class When_receiving_messages
             // setting up the environment
             var container = new TransportPartsContainer();
 
-            var topologySectionManagerInternal = await SetupEndpointOrientedTopology(container, settings, "sales");
+            settings.Set<Conventions>(new Conventions());
+            var topology = new EndpointOrientedTopologyInternal(container);
+
+            topology.Initialize(settings);
+
+            var extensions = new TransportExtensions<AzureServiceBusTransport>(settings);
+            settings.SetDefault("NServiceBus.Routing.EndpointName", "sales");
+            extensions.NamespacePartitioning().AddNamespace("namespaceName", AzureServiceBusConnectionString.Value);
+
+            // create the topologySectionManager
+            var namespaceManagerCreator = new NamespaceManagerCreator(settings);
+
+            var namespaceManagerLifeCycleManagerInternal = new NamespaceManagerLifeCycleManagerInternal(namespaceManagerCreator);
+            var topologyCreator = new TopologyCreator(new AzureServiceBusSubscriptionCreatorV6(topology.Settings.SubscriptionSettings, settings), 
+                new AzureServiceBusQueueCreator(topology.Settings.QueueSettings,settings),  new AzureServiceBusTopicCreator(topology.Settings.TopicSettings), 
+                namespaceManagerLifeCycleManagerInternal);
+            var topologySectionManagerInternal = new EndpointOrientedTopologySectionManager(settings, container);
+            await topologyCreator.Create(topologySectionManagerInternal.DetermineResourcesToCreate(new QueueBindings()));
 
             // setup the operator
-            var clientEntities = new MessageReceiverLifeCycleManager(new MessageReceiverCreator(new MessagingFactoryLifeCycleManager(new MessagingFactoryCreator(new NamespaceManagerLifeCycleManagerInternal(new NamespaceManagerCreator(settings)), settings), settings), settings), settings);
+            var createMessagingFactoriesInternal = new MessagingFactoryCreator(namespaceManagerLifeCycleManagerInternal, settings);
+            var manageMessagingFactoryLifeCycleInternal = new MessagingFactoryLifeCycleManager(createMessagingFactoriesInternal, settings);
```

Comment:
`Internal` in variable
I'll pause here as there's quite a few of them assuming you'll just rename the rest 😺